### ISSUE TITLE
Reproduce on CIFAR-10 with WRN40x2

### ISSUE THREAD

**Issue by hankook** [kakaobrain/fast-autoaugment#23]
I tried the following script with the found policy on CIFAR10 as described in README.

```
python FastAutoAugment/train.py -c confs/wresnet40x2_cifar10_b512.yaml --aug fa_reduced_cifar10 --dataset cifar10
```

With the script, I achieved 92.89% accuracy (i.e., 7.11% error rate) on the test set. However, the 3.7% error is reported in README. I think this gap is too large, so it seems a bug.

How to fix it?

(My PyTorch version is 1.1.0 and machine has 4 Titan Xp GPUs.)

**Comment by hankook** [comment]
I fount that the issue comes from CosineAnnealingLR of PyTorch (1.1.0). After upgrading PyTorch (1.1.0->1.3.0), WRN40x2 achieves 3.5% error rate on CIFAR10 :)

**Comment by heartInsert** [comment]
> I fount that the issue comes from CosineAnnealingLR of PyTorch (1.1.0). After upgrading PyTorch (1.1.0->1.3.0), WRN40x2 achieves 3.5% error rate on CIFAR10 :)

Hi, mate,  what's  the  problem  about  CosineAnnealingLR  in  PyTorch (1.1.0) ?

**Comment by hankook** [comment]
@heartInsert As far as I can remember, the learning rate was decreasing very rapidly. It would be better to check the difference of the `CosineAnnealingLR.get_lr` function  between [1.1.0](https://pytorch.org/docs/1.1.0/_modules/torch/optim/lr_scheduler.html#CosineAnnealingLR) and [1.3.0](https://pytorch.org/docs/1.3.0/_modules/torch/optim/lr_scheduler.html#CosineAnnealingLR).
